PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -984,6 +984,7 @@ void WindowsDlg::putItemsToClipboard(bool isFullPath)
 	constexpr int pathColumn = 1;
 
 	TCHAR str[MAX_PATH] = {};
+	const generic_string crlf = _T("\r\n");

```

Comment:
I would say it could be CR, LF or CRLF - it depends on the current document's EOL (Windows, Unix or Macintosh)

Reply:
After testing, the pasted EOL will conform to the destination's EOL.
So you're right, let's make it as Windows format.